From d2e07e1324684e0f40be42c64870bf3b59489253 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <slomo@circular-chaos.org>
Date: Mon, 23 Sep 2013 12:40:22 +0200
Subject: [PATCH 1/2] Disable AC_C_ALWAYS_INLINE

It causes errors with the new Xcode 5.0 toolchain and is not necessarily
useful anyway
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 9f4831e..fd4196f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -142,7 +142,6 @@ AC_CHECK_GENERATE_INTTYPES([include])
 
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
-AC_C_ALWAYS_INLINE
 AC_C_RESTRICT
 AC_C_BUILTIN_EXPECT
 AC_C_BIGENDIAN
-- 
2.1.4

